pom.xml

```diff
@@ -204,17 +204,17 @@
     <connection>scm:git:git://github.com/jenkinsci/node-sharing-plugin.git</connection>
     <developerConnection>scm:git:git@github.com:jenkinsci/node-sharing-plugin.git</developerConnection>
     <url>https://github.com/jenkinsci/node-sharing-plugin</url>
-    <tag>parent-1.2.6</tag>
+    <tag>parent-1.2.9</tag>
   </scm>
 
   <properties>
-    <jenkins.version>2.60.3</jenkins.version>
-    <java.level>8</java.level>
-    <jenkins-test-harness.version>2.40</jenkins-test-harness.version>
+    <jenkins.version>2.332.4</jenkins.version>
+    <java.level>11</java.level>
+    <jenkins-test-harness.version>1922.v91779ee45b_5e</jenkins-test-harness.version>
```

Comment:
I suspect the reason the build fails with `Failed to find org-netbeans-insane-hook.jar` is the incompatibility between `maven-hpi-plugin` (here upgraded through `org.jenkins-ci.plugins:plugin`) and `jenkins-test-harness` (see https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L109).

The project is incorrectly hardcoding `jenkins-test-harness` version in the number of places[1]. Deleting those dependency declarations should cause the `org.jenkins-ci.plugins:plugin` to add it in the version declared by `<jenkins-test-harness.version>`.

Let's delete `<jenkins-test-harness.version>` from here entirely, so that we use the JTH version directly from plugin pom. We will add it IFF it turns out we need to override the version.

[1]
```
mvn dependency:tree | grep jenkins-test-harness:
[INFO] +- org.jenkins-ci.main:jenkins-test-harness:jar:2.56:test
[INFO] +- org.jenkins-ci.main:jenkins-test-harness:jar:2.56:test
[INFO] +- org.jenkins-ci.main:jenkins-test-harness:jar:2.56:test
[INFO] +- org.jenkins-ci.main:jenkins-test-harness:jar:2.56:test
[INFO] +- org.jenkins-ci.main:jenkins-test-harness:jar:2.56:test
```